### ISSUE TITLE
ImageDataGenerator.fit() not called

### ISSUE THREAD

**Issue by gabrieloexle** [mercury-ml-team/mercury-ml#18]
If the following parameters for augmentation are specified
`featurewise_center`
`featurewise_std_normalization`
`zca_whitening`
then after creating
`generator = ImageDataGenerator`
the method 
`generator.fit(data)`
should be called to calculate e.g. the `featurewise_std_normalization` once.
See first example of https://keras.io/preprocessing/image/
In mercury-ml, this is missing e.g. in 
common/providers/source_reading/disk.py line 58-71

So either the method fit() is called somewhere or these 3 specific parameters should be deprecated.

**Comment by gabrieloexle** [comment]
I created an example what's necessary for the normalization.
It can be found here:
https://github.com/gabrieloexle/mercury-ml/blob/feature_normalization/examples/keras/feature_normalization.py
